ProjectSettings/InputManager.asset

```diff
@@ -9,8 +9,8 @@ InputManager:
     m_Name: Horizontal
     descriptiveName: 
     descriptiveNegativeName: 
-    negativeButton: left
-    positiveButton: right
+    negativeButton: right
```

Comment:
This change here is going to cause errors for everyone who is using the Horizontal input. Do you think you could either live with it being as it was before, *or* could you invert it in your control script?